retail/interactive-tutorial/TestResourcesSetupCleanup/CreateTestResources.cs

```diff
@@ -323,10 +329,12 @@ public static void PerformCreationOfTestResources()
         // Create a GCS bucket with products.json file.
         var createdProductsBucket = CreateTestResources.CreateBucket(productsBucketName);
         CreateTestResources.UploadBlob(createdProductsBucket.Name, productFilePath, ProductFileName);
+        CreateTestResources.UploadBlob(createdProductsBucket.Name, invalidProductFilePath, InvalidProductFileName);
```

Comment:
Will this and the other Upload attempt fail? If the fails the script won't execute completely.

Reply:
No, there is just another another file in the bucket that has 1 invalid product. After products import in console output we have message about number of successfully imported products and number of failures during the importing. The same with  events.